app/script/conversation/ConversationRepository.coffee

```diff
@@ -2126,25 +2123,6 @@ class z.conversation.ConversationRepository
       return message_ets
 
   ###
-  Check for duplicates by event IDs and cache the event ID.
-
-  @private
-  @param message_et [z.entity.Message] Message entity
-  @param conversation_et [z.entity.Conversation] Conversation entity
-  @return [Boolean] Returns true if event is a duplicate
-  ###
-  _check_for_duplicate_event_by_nonce: (message_et, conversation_et) ->
```

Comment:
We have seen duplicate messages again. Are we sure we can remove this check and should not hook it up again?